plugins/notifications-node/src/service/DefaultNotificationService.ts

```diff
@@ -28,7 +28,16 @@ export type NotificationServiceOptions = {
 export type NotificationRecipients =
   | {
       type: 'entity';
+      /**
+       * Entity references to send the notifications to
+       */
       entityRef: string | string[];
+      /**
+       * Optional list of user references to filter out of the resolved recipients.
+       * Usually the currently logged-in user for preventing sending notification
+       * of user action to him/herself.
+       */
+      filterUsers?: string[];
```

Comment:
Naming here doesn't immediately communicate the intent I feel? What do you think about some like `excludeUserRefs`, `excludeEntityRefs`, `exclude: { entityRef: ... }`? My first hunch when seeing the code was actually that these were the only users to be notified

Reply:
Yeah I agree, the naming could be better. Actually this could also work for entityRefs with small modification so maybe `excludeEntityRefs`?